op-e2e/system/e2esys/setup.go

```diff
@@ -748,6 +754,10 @@ func (cfg SystemConfig) Start(t *testing.T, startOpts ...StartOption) (*System,
 		return nil, fmt.Errorf("waiting for blocks: %w", err)
 	}
 
+	if action, ok := parsedStartOpts.Get("afterL1Start", ""); ok {
```

Comment:
What is the purpose of this code? any comments?

Reply:
This code will execute Opts, which should be run after L1 is deployed and before the L2 chain starts. Like the batch inbox contract. Similar logic exists like `afterRollupNodeStart` and `beforeBatcherStart` in this Start function.